packages/lisk-core/src/helpers/logger.js

```diff
@@ -0,0 +1,6 @@
+const liskBunyan = require('@lisk/lisk-logger-bunyan');
+const config = require('../helpers/config');
```

Comment:
This config is dynamically generated, with after a lot of computation. This may involve parsing different command line parameters or EVN variables. The final parsed configuration is available only on the top level, which should and kept as immutable reference there. I don't think `require('../helpers/config');` is going to solve all that mess. 

So if we have to pass that configuration from top level to logger, we can't simply use below to get it work.

```
const logger = require('./helpers/logger');
``` 

We need to require and initiate with configuration, and for that purpose factories are good or constructor functions, but such helper files will not solve the problem.